src/components/Settings/preferences/accordions/MonthlyGoalAccordion/MonthlyGoalAccordion.tsx

```diff
@@ -135,7 +135,7 @@ export const MonthlyGoalAccordion: React.FC<MonthlyGoalAccordionProps> = ({
       onAccordionChange={handleAccordionChange}
       expandedPanel={expandedPanel}
       label={label}
-      value={formattedMonthlyGoal}
+      value={formattedMonthlyGoal || formattedCalculatedGoal}
```

Comment:
Should it mention that it's using the calculated goal. USers won't know unless they click into the accordion.